openapi.json

```diff
@@ -23,6 +23,17 @@
         },
         "operationId": "getRecommendationList",
         "parameters": [
+          {
```

Comment:
I am not sure I understand the need for introducing an additional request parameter to control the response format. Could we instead rely on the standard Accept header to determine whether the client expects JSON or CSV? This would follow HTTP best practices (https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Accept) and simplify the API surface for users.

The handler can inspect the Accept header and respond accordingly by setting the appropriate `Content-Type` (which you already set). Is there a specific reason we need the custom parameter?

Reply:
It is a common practice to use query parameters for such use cases, because it provides a direct link for the user `/data?format=csv`.
Additionally, the Accept header is useful for direct API users, whereas for browsers and simple clients the query parameters are a bit more straight-forward.
I think we should add the Accept header as an enhancement, could you create a card for the same and assign it to me?